config.py

```diff
@@ -92,6 +92,80 @@ def load(self):
 The head of the HAProxy config. This contains global settings
 and defaults.
 '''))
+        self.add_template(
```

Comment:
How about we truncate the map every time MLB is started? This will address the scenario you pointed out, and it will simplify the logic.